docs/api/main-config-typescript.md

```diff
@@ -6,15 +6,32 @@ Parent: [main.js|ts configuration](./main-config.md)
 
 Type:
 
+<IfRenderer renderer='react'>
+
 ```ts
 {
   check?: boolean;
   checkOptions?: CheckOptions;
   reactDocgen?: 'react-docgen' | 'react-docgen-typescript' | false;
   reactDocgenTypescriptOptions?: ReactDocgenTypescriptOptions;
+  skipCompiler?: boolean;
 }
 ```
 
+</IfRenderer>
+
+<IfRenderer renderer={['angular', 'vue', 'web-components', 'ember', 'html', 'svelte', 'preact', 'qwik','solid' ]}>
```

Comment:
FYI, you can do this instead:

```suggestion
<If notRenderer="react">
```

Reply:
"Docs":  https://main--5be26744d2f6250024a9117d.chromatic.com/?path=/story/screens-docsscreen-if--single-exclude-is-visible